README.md

```diff
@@ -5,3 +5,128 @@
+```
+
+In this case, there is some "block" (section, idea, maybe component) named `new-card-form`, and this style is for the "element" that represents the header by name of `header`. This style makes the header in the new card form `text-align: center;`. Note that this does not dictate what kind of HTML tag this is, or what its parent or children are.
+
+Again, please limit the time you spend on styling, and reach out often and frequently.
+
+## Requirements
+
+### Wave 0: Establish the Relationships
+
+1. Make a diagram with all of the components, illustrating which components are nested within each other, and making it clear which components are siblings to each other.
+1. For each component, list out what data needs to be referenced within it, and what data type each piece of data is. Particularly, think about:
+    - If the `PlayerSubmissionForm` component handles a player's submission of poetry, what should that component do with that data?
+    - How should the `FinalPoem` component get all of the parts of the poem?
+    - What kind of input elements should `PlayerSubmissionForm` use? What does this mean about the `PoemInput` component?
+    - What should the `Game` component represent and be responsible for?
+1. For each piece of data in each component, determine if it should be part of that component's `state` or passed in as a `prop`.
+
+The waves are organized in this order:
+
+1. Set up the `PlayerSubmissionForm` component with a hard-coded form and prove that the `Game` component can receive submission data from the form when the form is submitted
+1. Set up the `FinalPoem` component so that it shows all of the lines of poetry that the players have previously submitted
+1. Work on details that show, hide, or modify different components: showing the most recent submission or conditionally showing the final poem
+1. Refactor the hard-coded `PlayerSubmissionForm` so it dynamically generates the form
+
+### Wave 1: Let Players Submit Lines of Poetry
+
+The goal of this wave is to set up the relationship between the `Game` component and the `PlayerSubmissionForm`. The `PlayerSubmissionForm` should render a set of inputs for the given poetry format, and submitting the form should send the form data back up to the `Game` component.
+
+Implement this first iteration of the `PlayerSubmissionForm` with a set of hard-coded inputs: the ones that represent the absolute format "The adjective noun adverb verb the adjective noun" (one for each adjective, noun, adverb, and verb, for six total).
+
+#### User Stories
+
+- As a player, I want to be able to fill out the six text input fields that mimic the poetry format and press a submit button, so that I fill out the line of poetry.
+- As a player, I want to be able to see placeholder text in each text input that describes what kind of word I should fill in.
+- As a player, I want the `Game` component to have the data of my submission, so that the `Game` component keeps track of all of the submissions.
```

Comment:
I think this is helpful for students to figure out how it's working, but user stories don't usually describe implementation details, just user-perspective input and output.

Reply:
Yeah I definitely agree... Mostly added this because I wasn't sure how to turn it into a User Story when this piece of implementation I wanted for this wave doesn't actually have a good way to be represented to the user until the next wave